cmd/kube-apiserver/app/server.go

```diff
@@ -277,6 +279,10 @@ func Run(s *options.ServerRunOptions) error {
 	genericConfig.EnableOpenAPISupport = true
 	genericConfig.EnableMetrics = true
 	genericConfig.OpenAPIConfig.SecurityDefinitions = securityDefinitions
+	genericConfig.LongRunningFunc = filters.BasicLongRunningRequestCheck(
```

Comment:
proxy/attach/exec/portforward as subresources are only likely to make sense to the apiserver... baking those into genericapiserver didn't seem useful